contracts/test/upgrade/MigrationSteps.t.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity =0.8.18;
+
+import {Test} from "forge-std/Test.sol";
+
+// NOTE TO DEVELOPERS:
+// This checklist test is used to ensure safety of the migration process.
+// These tests serve as a guide to the migration process, step by step.
+// Each step should be marked as complete switching `false` to `true` in `assertTrue`.
+contract MigrationStepsTest is Test {
+    function testStep0_deployContracts() public {
+        // Deploy contracts using CoreDeploy.s.sol and set the contract addresses in:
+        // `UpgradePreparation` and `test_MIGRATION_WARNING_deployedContracts`
+        // Set `triggerMigrationWarning_setDeployedContracts` in `setDeployedContract` to `false` upon completion
+        // Run `test_MIGRATION_WARNING_deployedContracts` and `test_MIGRATION_WARNING_deployedContractsBehavior` to confirm this step is over.
+        assertTrue(false);
+    }
+
+    function testStep1_setFinalTokenUnits() public {
+        // By step 1, it is assumed that all the funds have been transferred to the multisig.
+        // Update InitialBountyHelper's token's list to reflect the final bounty units.
+        // Manually check the multisig on Etherscan to make sure multisig holds all funds
+        // Set `triggerMigrationWarning_finalTokens` in `Config` to `false` upon completion
+        // Run `test_MIGRATION_WARNING_finalizedTokens` to confirm this step is over.
+        assertTrue(false);
+    }
+
+    function testStep2_stopMockingSafeBalances() public {
+        // Remove `mockSafeBalances` from `UpgradePreparation`
+        // Run `test_MIGRATION_WARNING_safeBalances` to confirm this step is over.
+        assertTrue(false);
+    }
+
+    function testStep3_removeForkBlock() public {
+        // Remove `fork` from `UpgradePreparation`.
+        // NOTE: Foundry may cache forked state. Use `forge clean` every time you want to run a clean fork.
+        // run `test_MIGRATION_WARNING_forkBlock` to confirm this step is over.
+        assertTrue(false);
+    }
+
+    function testStep4_updateCurrentPrice() public {
+        // Update `MIGRATION_WARNING_getCurrentPrice` in `MigrationChecklist` to reflect the latest prices.
+        // Set `triggerMigrationWarning_getCurrentPrice` in `MigrationChecklist` to `false` upon completion
+        // Run `test_MIGRATION_WARNING_getCurrentPrice` to confirm this step is over.
```

Comment:
What is the `MIGRATION_WARNING_getCurrentPrice(address(0));` in this test supposed to be testing?

Reply:
apparently nothing. removed